lldb/source/Plugins/ObjectFile/PECOFF/ObjectFilePECOFF.cpp

```diff
@@ -355,11 +355,10 @@ size_t ObjectFilePECOFF::GetModuleSpecifications(
 }
 
 bool ObjectFilePECOFF::SaveCore(const lldb::ProcessSP &process_sp,
-                                const lldb_private::FileSpec &outfile,
-                                lldb::SaveCoreStyle &core_style,
+                                lldb_private::CoreDumpOptions &core_options,
                                 lldb_private::Status &error) {
-  core_style = eSaveCoreFull;
-  return SaveMiniDump(process_sp, outfile, error);
+  core_options.SetCoreDumpStyle(lldb::eSaveCoreFull);
```

Comment:
This should not set things to emit full core file all of the time. It used to be the only thing that the minidump core file supported, but that isn't true anymore. So this shouldn't muck with the settings. So don't set the style to full here.